landlab/bmi/bmi_bridge.py

```diff
@@ -319,11 +352,20 @@ def initialize(self, config_file):
             self._clock = TimeStepper(**clock_params)
 
             self._base = self._cls(grid, **params.pop(snake_case(cls.__name__), {}))
+            self._base.grid.at_node["boundary_condition_flag"] = self._base.grid.status_at_node
 
         def update(self):
             """Update the component one time step."""
             if hasattr(self._base, "update"):
                 self._base.update()
+            elif hasattr(self._base, "run_one_step"):
+                nargs = len(inspect.signature(self._base.run_one_step).parameters)
+
+                if nargs == 0:
+                    self._base.run_one_step()
+                else:
+                    self._base.run_one_step(self._clock.step)
+
```

Comment:
Does it make sense to make an issue about this (and tag LL2.0) requesting that all R1S methods take dt as the first (and only) argument...?

Reply:
Done.